.test/config-sra/config.yaml

```diff
@@ -48,7 +48,8 @@ calling:
       present:
         varlociraptor: 
           - "PRESENT"
-        filter: moderate
+        filter:
+          - moderate
```

Comment:
For convenience, it should still be possible to put just a string here without a list. Can you keep it that way for one of the tests?

Reply:
Good point. I just reversed one test to cover this.